src/bp/index.ts

```diff
@@ -55,11 +55,11 @@ process.PROJECT_LOCATION = process.pkg
 
 process.stderr.write = stripDeprecationWrite
 
-process.on('unhandledRejection', err => {
-  global.printErrorDefault(err)
+process.on('unhandledRejection', (reason: {} | null | undefined, promise: Promise<any>) => {
+  global.printLog([reason, promise])
```

Comment:
Although `global.printLog` seems to be available (because it is global), it is only being initialized later in `bootstrap.ts`, which is also where the `global.printErrorDefault` is being overridden to use the logger's name.  

This would prevent catching errors between the startup & inversify setting up